docs/lake/design.md

```diff
@@ -103,37 +108,36 @@ a "table" as all Zed data is _self describing_ and can be queried in a
 schema-agnostic fashion.  Data of any _shape_ can be stored in any pool
 and arbitrary data _shapes_ can coexist side by side.
 
-### Scan
-
-Data is read from a pool with the `scan` command.  By default, `scan`
-generates sorted output of all of the pool data in the configured key order.
-The order can be overridden with `-order asc` or `-order desc`.
-
-A range can be specified with `-from` or `-to`.
+### Query
 
-The default output format is ZNG though it can be overridden with the various
-supported output formats.
+Data is read from a pool with the `query` command.
+A range can be specified with `-from` and/or `-to`.
+The default output format is ZNG though this can be overridden with `-f`
+to specify one of the various supported output formats.
 
 This example reads every record from the `logs` pool starting
-from time `2020-1-1T12:00`, in ascending order,
-sends the  results as ZNG to stdout, then pipes the output to `zq` to count the records:
+from time `2020-1-1T12:00` and sends the results as ZSON to stdout.
 ```
-zed lake scan -p logs -from 2020-1-1T12:00 -order asc | zq -z "count()" -
+zed lake query -p logs -f zson -from 2020-1-1T12:00
 ```
-
-### Query
-
-Of course, the example is an inefficient way to count records.  Instead of
-reading the records out of the lake and processing them, a better approach
-is to push the query into the lake.
-
-The `query` command lets you do this, e.g.,
+A much more efficient format for transporting query results is the
+row-oriented, compressed, and binary format ZNG.  Because ZNG
+streams are easily merged and composed, a query in ZNG format
+from a pool can be can be piped to another `zed query` instance, e.g.,
+```
+zed lake query -p logs -f zng -from 2020-1-1T12:00 | zed query -f table "count() by query"
```

Comment:
I think these `zed lake query` and `zed query` examples would be clearer if the Zed query itself didn't contain "query".
```suggestion
zed lake query -p logs -f zng -from 2020-1-1T12:00 | zed query -f table "count() by field"
```